tests/widgets/drive.spec.coffee

```diff
@@ -1,19 +1,2 @@
 m = require('mochainon')
```

Comment:
I think you can remove this file altogether. It'd be very difficult to test the actual visual part.